KP5-Topic2/KP1-Topic2/target/surefire-reports/com.aguirre.KP1_Topic2.AppTest.txt

```diff
@@ -0,0 +1,4 @@
+-------------------------------------------------------------------------------

```

Comment:
Please include the folder target in .gitignore file to avoid to upload to the repo these generated files